### PR TITLE
DIV-1020: Update Initial Filing page front-end

### PR DESCRIPTION
**Pull request by ariannedee**

### CODE REVIEW

**Comment by ariannedee** [comment]
@molund FYI, the logic of which files need to be uploaded is in `cso_filing.py` (only initial filing has been done so far). You can use that to determine which PDFs need to be sent to CSO (DIV-1184). In addition, there are a few that don't go through the uploader and are sent directly from what we generate, which can just be another function in `cso_filing.py`.

File: edivorce/apps/core/templates/dashboard/help_scanning.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+{% block content %}
+    <h1>Get Help Opening and Saving PDF forms</h1>
+    <div>
+        <h3>Your divorce forms</h3>
+
+        <p>You can review, print, or save your forms by clicking on the
+            &ldquo;Review and Print&rdquo; button next to each form. This will open or
+            download a PDF version of the form.</p>
+
+        <p>&ldquo;PDF&rdquo; stand for Portable Document Format, which is a file
```
Comment (molund) [inline edivorce/apps/core/templates/dashboard/help_scanning.html:18]: stand**s** for (missing s)

File: edivorce/apps/core/templates/dashboard/help_scanning.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+            <a href="https://helpx.adobe.com/acrobat/using/display-pdf-in-browser.html"
+               target="_blank">Adobe’s instructions for displaying PDFs</a> in
+            various browsers for detailed instructions.
+        </p>
+    </div>
+
+    <div class="question-well">
+        <h3>Save the forms to your computer</h3>
+
+        <p>It's a great idea to save the PDF’s of your forms to your computer so
```
Comment (molund) [inline edivorce/apps/core/templates/dashboard/help_scanning.html:59]: remove the apostrophe

File: vue/src/pages/initial-filing/InitialFiling.vue [hunk 3]
```diff
@@ -145,19 +17,18 @@
       Uploader,
     },
     props: {
-      signingLocation: String,
-      signingLocationYou: String,
-      signingLocationSpouse: String,
+      formTypes: JSON,
```
Comment (molund) [inline vue/src/pages/initial-filing/InitialFiling.vue:20]: I'm not sure if JSON is a valid type for props.  Did you try "Object"?

**Review by molund — APPROVED** [review]

File: edivorce/apps/core/templates/dashboard/help_scanning.html (hunk 1 above)
Comment (ariannedee) [reply edivorce/apps/core/templates/dashboard/help_scanning.html:18]: Oh yeah, this is a dummy page and I will fill it in later. But good catch. The content is from another page in the app ("Help saving PDF")

**Pull request merged**